Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -276,6 +281,21 @@ export class AssetManager {
               packageVersion: version,
             },
           } as BotTemplate;
+          if (isArray(keywords)) {
+            if (keywords.indexOf('bf-dotnet-functions') != -1 || keywords.indexOf('bf-dotnet-webapp') != -1) {
+              templateToReturn.dotnetSupport = {
+                functionsSupported: keywords.indexOf('bf-dotnet-functions') != -1,
+                webAppSupported: keywords.indexOf('bf-dotnet-webapp') != -1,
+              };
+            }
+            if (keywords.indexOf('bf-js-functions') != -1 || keywords.indexOf('bf-js-webapp') != -1) {
+              templateToReturn.nodeSupport = {
+                functionsSupported: keywords.indexOf('bf-js-functions') != -1,
+                webAppSupported: keywords.indexOf('bf-js-webapp') != -1,
```

Comment:
Should be able to use `keywords.includes` here instead of indexOf.

Reply:
Good catch, I have changed it to use .includes()